holoviews/plotting/mpl/__init__.py

```diff
@@ -157,7 +158,8 @@ def grid_selector(grid):
 
 
 MPLPlot.sideplots.update({Histogram: SideHistogramPlot,
-                          GridSpace: GridPlot})
+                          GridSpace: GridPlot,
+                          Spikes: MarginalRugPlot})
```

Comment:
Do we need the word ``Marginal``? That depends on whether ``RugPlot`` can be used without being a side plot or not. If it is always a side-plot, ``RugPlot`` would do...